wisdem/glue_code/glue_code.py

```diff
@@ -432,42 +432,35 @@ def setup(self):
-        self.connect('nacelle.shrink_disc_mass',   'drivese.shrink_disc_mass')
-        self.connect('nacelle.carrier_mass',       'drivese.carrier_mass')
-        self.connect('nacelle.flange_length',      'drivese.flange_length')
-        self.connect('nacelle.gearbox_input_xcm',  'drivese.gearbox_input_xcm')
-        self.connect('nacelle.hss_input_length',   'drivese.hss_input_length')
-        self.connect('nacelle.distance_hub2mb',    'drivese.distance_hub2mb')
-        self.connect('nacelle.yaw_motors_number',  'drivese.yaw_motors_number')
-        self.connect('nacelle.gearbox_efficiency', 'drivese.gearbox_efficiency')
-        self.connect('nacelle.generator_efficiency','drivese.generator_efficiency')
-        self.connect('tower.diameter',             'drivese.tower_top_diameter', src_indices=[-1])
+            self.connect('assembly.rotor_diameter',    'drivese.rotor_diameter')
+            self.connect('configuration.upwind',       'drivese.upwind')
+            self.connect('configuration.direct_drive', 'drivese.direct_drive')
+            self.connect('configuration.n_blades',     'drivese.n_blades') 
+            self.connect('sse.powercurve.rated_Omega', 'drivese.generator.n_nom')
+            self.connect('sse.powercurve.P_aero',      'drivese.generator.P_mech', src_indices=[-1])
+            self.connect('sse.powercurve.rated_Q',     'drivese.generator.T_rated')
+            self.connect('sse.gust.V_gust',            'drivese.spinner.gust_ws')
+            self.connect('control.rated_power',        'drivese.machine_rating')    
+            self.connect('nacelle.uptilt',             'drivese.tilt')
+            self.connect('tower.diameter',             'drivese.D_top', src_indices=[-1])
+            if modeling_options['Analysis_Flags']['OpenFAST'] and modeling_options['openfast']['analysis_settings']['Analysis_Level'] == 2:
+                self.connect('aeroelastic.Fxyz', 'drivese.F_hub')
+                self.connect('aeroelastic.Mxyz', 'drivese.M_hub')
+                self.connect('aeroelastic.Mxyz', 'drivese.hub_shell.max_torque', src_indices=[0])
+                self.connect('aeroelastic.My',   'drivese.pitch_system.BRFM')
+            else:
+                self.connect('rlds.aero_hub_loads.Fxyz_hub_aero', 'drivese.F_hub')
+                self.connect('rlds.aero_hub_loads.Mxyz_hub_aero', 'drivese.M_hub')
+                self.connect('rlds.aero_hub_loads.Mxyz_hub_aero', 'drivese.hub_shell.max_torque', src_indices=[0])
+                self.connect('rlds.frame.root_M',                 'drivese.pitch_system.BRFM', src_indices=[1])
+            self.connect('elastic.precomp.blade_mass', 'drivese.pitch_system.blade_mass')
+            self.connect('blade.pa.chord_param',       'drivese.blade_root_diameter', src_indices=[0])
 
         # Connections to TowerSE
         if modeling_options['flags']['tower']:
-            self.connect('drivese.top_F',                 'towerse.pre.rna_F')
```

Comment:
What changed between `top` and `base` here? Is this a meaningfully different value or just a different name for the same value as before?